sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ArithmeticExpressionSuite.scala

```diff
@@ -69,6 +69,7 @@ class ArithmeticExpressionSuite extends SparkFunSuite with ExpressionEvalHelper
 
     checkEvaluation(-c1, -1.1, row)
     checkEvaluation(c1 + c2, 3.1, row)
+    checkDoubleEvaluation(Rand(30), (0.7363714192755834 +- 0.001), row)
```

Comment:
I am in Beijing now.
Except for difficulty of accessing gmail, github is quite slow as well.